### DISCUSSION TITLE
checking examples (brats task 001 ipynb) again

### DISCUSSION THREAD

**Discussion by bhralz** [Project-MONAI/MONAI#7538] [General]
Hello,
After installing latest version of Monai[all], I think the example brats task001 dont work correctly.

Is it possible to check it and bug fixing?

thanks

**Comment by KumoLiu** [comment]
Hi @bhralzz, thanks for the report.
Could you please share more information? Such as the command you install the MONAI, and the whole error log you get from which notebook.
Thanks.

**Comment by bhralz** [comment]
@KumoLiu 
thanks for your response:


goal is to rerun brats_segmentation_3d

os : win10

command tested for monai installation: pip install monai[all]

test on python versions: 3.8, 3.9, 3.10

AttributeError: module 'numpy' has no attribute 'bool'.

by changing torch.bool: np.bool_ in misc.py or by donwgrading numpy version it changed to the below errors:

cannot import name 'Invertd' from 'monai.transforms'

**Comment by KumoLiu** [comment]
I didn't find where we use `np.bool` in the source code.
Could you please refer to the install guide to install and see whether the monai can be imported successfully.
https://docs.monai.io/en/latest/installation.html

**Comment by bhralz** [comment]
**the source of error is:**

File C:\ProgramData\anaconda3\envs\b8\lib\site-packages\monai\utils\misc.py:266
    torch.bool: np.bool


**another test based on installation.html verification of install:
the result of execution of
import monai
monai.config.print_config()**

C:\Users\BHR\.conda\envs\b9\lib\site-packages\monai\utils\misc.py:266: FutureWarning: In the future `np.bool` will be defined as the corresponding NumPy scalar.
  torch.bool: np.bool,
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\BHR\.conda\envs\b9\lib\site-packages\monai\__init__.py", line 36, in <module>
    from .utils.module import load_submodules  # noqa: E402
  File "C:\Users\BHR\.conda\envs\b9\lib\site-packages\monai\utils\__init__.py", line 16, in <module>
    from .misc import *
  File "C:\Users\BHR\.conda\envs\b9\lib\site-packages\monai\utils\misc.py", line 266, in <module>
    torch.bool: np.bool,
  File "C:\Users\BHR\.conda\envs\b9\lib\site-packages\numpy\__init__.py", line 353, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

**Comment by KumoLiu** [comment]
Perhaps upgrade your MONAI version.
https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/misc.py
https://github.com/Project-MONAI/MONAI/blob/95f69dea3d2ff9fb3d0695d922213aefaf5f0c39/requirements.txt#L2
Thanks.

**Comment by bhralz** [comment]
by saving torch and monai[all] updated the below error emerged. there is a conflict of version

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pytorch-ignite 0.4.2 requires torch<2,>=1.3, but you have torch 2.2.1 which is incompatible.
torchvision 0.14.1 requires torch==1.13.1, but you have torch 2.2.1 which is incompatible.


is it possible to serve step by step installation command for monai[all] in windows 10?

**Comment by KumoLiu** [comment]
Hi @bhralzz, 
I think we set `pytorch-ignite==0.4.11` in requirement and didn't fixed the version of torchvision 
https://github.com/Project-MONAI/MONAI/blob/95f69dea3d2ff9fb3d0695d922213aefaf5f0c39/requirements-dev.txt#L3
https://github.com/Project-MONAI/MONAI/blob/95f69dea3d2ff9fb3d0695d922213aefaf5f0c39/requirements-dev.txt#L27

**Comment by bhralz** [comment]
whats your idea to rerun the example?
Should I follow a step by step guide? in installation or need to modify code? any requirements list?...

**Comment by KumoLiu** [comment]
Hi @bhralzz, here is the installation guide https://docs.monai.io/en/latest/installation.html.
What I suggest is that create a new virtual env or using docker.
Hope it helps, thanks.

**Reply by bhralz** [reply]
thanks
